config/environments/development.rb

```diff
@@ -74,4 +74,6 @@
     Bullet.console = true
     Bullet.rails_logger = true
   end
+
+  config.active_job.queue_adapter = :delayed_job
```

Comment:
should this be set in test and production too?

Reply:
Ga'h good catch - this was debug code slipping through

Reply:
Ok, now fixed